gomock/controller_test.go

```diff
@@ -132,7 +133,7 @@ func (s *Subject) BarMethod(arg string) int {
 	return 0
 }
 
-func (s *Subject) VariadicMethod(arg ...string) {}
+func (s *Subject) VariadicMethod(arg int, vararg ...string) {}
```

Comment:
What does this change do?

Reply:
To reproduce the original panic I needed a func that took in an argument of a different type than the vararg argument.